Content.Server/GameTicking/GameTicker.GamePreset.cs

```diff
@@ -43,7 +43,17 @@ void FailedPresetRestart()
             if (_configurationManager.GetCVar(CCVars.GameLobbyFallbackEnabled))
             {
                 ClearGameRules();
-                SetGamePreset(_configurationManager.GetCVar(CCVars.GameLobbyFallbackPreset));
+                var fallbackPreset = _configurationManager.GetCVar(CCVars.GameLobbyFallbackPreset);
+                if (fallbackPreset != "")
+                    SetGamePreset(fallbackPreset);
+                else if (startAttempt.Players.Length >= 5)
+                {
+                    SetGamePreset("traitor");
+                }
+                else
+                {
+                    SetGamePreset("extended");
+                }
```

Comment:
This defeats the point of GameLobbyFallbackEnabled and hardcodes gamemodes and timing in the ticker, a much cleaner solution would be making GameLobbyFallbackPreset an array and trying to set the preset in that order, i.e, array is ["traitor", "extended"] so it tries to set traitor and if it fails it tries extended

Reply:
Where should this array be stored?

Reply:
you should be able to just change the cvar def to an array